File: Python/1012.py
# -*- coding: utf-8 -*-

value = input().split(' ')


A = float(value[0])
B = float(value[1])
C = float(value[2])

pi = 3.14159

triangle = (A * C)/2 # The area of the rectangled triangle that has base A and height C
circle = pi * (C**2) # The area of the radius's circle C. (pi = 3.14159)
trapezium = (A+B) * (C/2) # The area of the trapezium which has A and B by base, and C by height.
square = B**2 # The area of ​​the square that has side B.
rectangle = A*B # The area of the rectangle that has sides A and B.



print("TRIANGULO: {:.3f}".format(triangle)) 
print("CIRCULO: {:.3f}".format(circle)) 
print("TRAPEZIO: {:.3f}".format(trapezium)) 
print("QUADRADO: {:.3f}".format(square)) 
print("RETANGULO: {:.3f}".format(rectangle))

File: JavaScript/1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

let pi = 3.14159;
let A = parseFloat(lines[0]);
let B = parseFloat(lines[1]);
let C = parseFloat(lines[2]);

console.log(`TRIANGULO: ${((A*C)/2).toFixed(3)}`);
console.log(`CIRCULO: ${(pi*(C**2)).toFixed(3)}`);
console.log(`TRAPEZIO: ${((A+B)*(C/2)).toFixed(3)}`);
console.log(`QUADRADO: ${(B**2).toFixed(3)}`);
console.log(`RETANGULO: ${(A*B).toFixed(3)}`);

/*var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

const pi = 3.14159;
let A = parseFloat(lines[0]);
let B = parseFloat(lines[1]);
let C = parseFloat(lines[2]);

function  areaRecTriangle()
{
    console.log(`TRIANGULO: ${((A*C)/2).toFixed(3)}`);
};
function  areaCircle()
{
    console.log(`CIRCULO: ${(pi*(C**2)).toFixed(3)}`);
};
function  areaTrapezium()
{
    console.log(`TRAPEZIO: ${((A+B)*(C/2)).toFixed(3)}`);
};
function  areaSquare()
{
    console.log(`QUADRADO: ${(B**2).toFixed(3)}`);
};
function  areaRectangle()
{
    console.log(`RETANGULO: ${(A*B).toFixed(3)}`);
};

areaRecTriangle();
areaCircle();
areaTrapezium();
areaSquare();
areaRectangle();
*/